spec/person_spec.rb

```diff
@@ -0,0 +1,47 @@
+require_relative '../person'
+require_relative '../rental'
+require_relative '../book'
+
+describe 'Person object' do
+  context 'should create an instance of Person class' do
+    person = Person.new(15, 'John', 'true')
+
+    it 'returns a Person object' do
+      expect(person.class) == Person
+    end
+
+    it 'returns correct names of person object instance variables' do
+      expect(person.age).to eq 15
+      expect(person.name).to eq 'John'
+      expect(person.parent_permission).to eq 'true'
```

Comment:
- [ ] your tests are failing because it is complaining of this line, kindly fix this
![image](https://user-images.githubusercontent.com/66097371/194333389-9fbb436f-df49-4d8c-8c69-c8bb756a715d.png)